docs/themes/porter/static/src/sass/helm-home.scss

```diff
@@ -56,7 +56,7 @@ nav.home-nav {
 
     img,
     svg {
-      min-height: 150px;
+      // min-height: 150px;
```

Comment:
There is still a css rule that is making the logo stretched on the homepage

```css
nav.home-nav h1 img, nav.home-nav h1 svg {
    min-height: 150px;
}
```

Reply:
looking into this now